trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/induced_drag_aircraft.py

```diff
@@ -133,6 +133,7 @@ def induced_drag_aircraft(state,settings,geometry):
         for wing in wings:
             if wing.areas.reference>S:
                 AR = wing.aspect_ratio
+                S  = wing.areas.reference 

```

Comment:
This doesn't look like it adds anything. If it is not used, remove it

Reply:
No this is a necessary line that was missing. This looks for the largest wing and decides it's the main wing to use as the AR. The S helps tracks the wings aspect ratio.

See #387

Reply:
Ahhh ok. Thanks for clarifying.